designs/SPEC_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+
+# Spec Name
+
+* Status: In Review / Reviewed / Implementing / Implemented
+* Author(s): [NuGet](https://github.com/NuGet)
+* Issue: [1](https://github.com/NuGet/Home/issues/1) Issue Title
+* Type: Feature

```

Comment:
what other types might there be?  Bug fixes shouldn't need a design document. FOr something big like CPVM, where there's an overarching design doc, but then smaller more specific design docs, do we use different values here?  Or maybe just remove this field?

Reply:
Great point. I'll remove it.
Think I had something else in mind when i did this :D

Sometimes, even big bugs lead to a refactoring that might require a spec, but it's really a DCR then. 

It really provides no value, so removing is definitely the best.